core/src/test/java/org/hibernate/ogm/backendtck/queries/SimpleQueriesTest.java

```diff
@@ -483,7 +485,7 @@ public void testAutoFlushIsAppliedDuringQueryExecution() throws Exception {
 		hypothesis.setPosition( 29 );
 		session.persist( hypothesis );
 
-		if ( EnumSet.of( MONGODB, NEO4J ).contains( TestHelper.getCurrentDialectType() ) ) {
+		if ( EnumSet.of( GridModule.MONGODB, GridModule.NEO4J ).contains( TestHelper.getCurrentModule() ) ) {
```

Comment:
I thought the notion of "module" is not needed really, all tests can either be skipped / controlled by dialect or datastore provider?